### ISSUE TITLE
URL length limit

### ISSUE THREAD

**Issue by diatomsRcool** [jhpoelen/effechecka#56]
I'm bumping into the configured URL limit of 2053. Any chance of lengthening that? My longest test url is 4458.

**Comment by jhpoelen** [comment]
Yep, this is a limitation for the ease of usage of http get requests. Workarounds would include to use fewer decimals, or split the shapes into parts. A proper fix would be to implement a http post, where the body can be pretty big. Not hard to do, but it would require some time and testing.
